source/solvers/vof_subequations_interface.cc

```diff
@@ -56,10 +48,12 @@ VOFSubequationsInterface<dim>::VOFSubequationsInterface(
               this->pcout,
               p_triangulation,
               p_simulation_control,
-              this->multiphysics_interface,
               this);
         }
     }
+  // Set all solutions as valid to be able to attach solution to output when
+  // user requests output of the initial condition
+  reset_subequations_solutions_validity(true);
```

Comment:
Should we instead calculate them at the initial condition to start?

Reply:
I've added a condition in `set_initial_condition()` to solve them if an output is required and adjusted this function :)